argus/config_generator/windows/cb_init.py

```diff
@@ -36,9 +36,11 @@ class BasePopulatedCBInitConfig(base.BaseWindowsConfig):
     def __init__(self, client):
         super(BasePopulatedCBInitConfig, self).__init__(client)
 
-    def set_conf_value(self, name, value=""):
-        """Set a config value in default section."""
-        self.conf.set("DEFAULT", name, value)
+    def set_conf_value(self, name, value="", section="DEFAULT"):
+        """Set a config value in the specified section."""
+        if not self.conf.has_section(section) and section != "DEFAULT":
```

Comment:
Also checking if the section is not `DEFAULT` , because of the behavior of the `has_section` method: `The DEFAULT section is not acknowledged.`